src/NDSCart.cpp

```diff
@@ -58,6 +58,9 @@ u32 CartID;
 bool CartIsHomebrew;
 bool CartIsDSi;
 
+struct NDSHeader Header;
```

Comment:
writing struct before a struct type shouldn't be necessary in C++, unless it's different with packed one?

Reply:
shrug idk how all this struct shit works and why everybody does it differently